.env

```diff
@@ -12,4 +15,10 @@ MAX_SEND=3000000
 COST_SOFT_LIMIT=100000
 COST_HARD_LIMIT=1000000
 REQUEST_SLEEP=100
-INITIAL_CONCURRENT=10
\ No newline at end of file
+INITIAL_CONCURRENT=10
+
+# Gaze's Indexer Network
+INDEXER_REPORTING_URL="https://indexer.api.gaze.network" # [Required] Indexer reporting URL, should not be changed.
+INDEXER_REPORTING_NAME="gaze-electrumx-indexer" # [Required] Indexer name
+INDEXER_REPORTING_WEBSITE_URL= # [Optional] Your official website
+INDEXER_REPORTING_INDEXER_API_URL= # Optional Your indexer API URL
```

Comment:
Wonder if we should rename everything around this to something like GAZE_NETWORK_REPORTING instead of INDEXER_REPORTING? What do you think? @gazenw

Reply:
Interesting, this makes it easier to understand.